src/content/en/updates/2011/11/Quota-Management-API-Fast-Facts.md

```diff
@@ -8,46 +8,45 @@ book_path: /web/updates/_book.yaml
 # Quota Management API : Fast Facts {: .page-title }
 
 {% include "web/_shared/contributors/agektmr.html" %}
+November 2011
 
-
-There’s various offline related features introduced to modern browsers through HTML5. While offline is convenient, its concept of quota has been left untouched for a long time. The latest version of Chrome browser has the first concept and its implementation of *Quota Management API*. It handles quota for AppCache, IndexedDB, WebSQL and File System API. Here’s a list of things you should keep in mind when working with Quota Management API in the latest Chrome.
+HTML5 introduced various offline-related features to modern browsers. While offline is convenient, its concept of quota has been left untouched for a long time. The latest version of Chrome browser has the first concept and its implementation of *Quota Management API*. It handles quota for AppCache, IndexedDB, WebSQL and File System API. Here’s a list of things you should keep in mind when working with Quota Management API in the latest Chrome.
```

Comment:
'latest version of Chrome browser'/'latest Chrome'

Never use these phrases. They can easily be misinterpreted (especially with text written in 2011). Always use as specific version number, Chrome 13, Chrome 58, whatever.